pipeline_dp/pipeline_operations.py

```diff
@@ -86,6 +86,14 @@ def reduce_accumulators_per_key(self, col, stage_name: str):
         """
         pass
 
+    @abc.abstractmethod
+    def flatten(self, col1, col2, stage_name: str):
```

Comment:
The problem is that there are other PipelineOperations, and w/o implementaiton this method in them, they are abstract clasess which can't be created.

Please add this method in other PipelineOperations:

In MultiProcessPipelineOperations please use implementation from LocalPipelineOpeations

In SparkPipelineOperations please add this method with implementation
`raise NotImplementedException("...") `
I'll implement it
In Spark please